lapis-docs/astro.config.mjs

```diff
@@ -224,7 +224,5 @@ export default defineConfig({
     },
     base: process.env.BASE_URL,
     site: process.env.ASTRO_SITE,
-    server: {
-        allowedHosts: true,
-    },
+    server: {},
```

Comment:
Why did you remove this? We recently added it to resolve https://github.com/GenSpectrum/LAPIS/issues/1124.

Reply:
The current set up throws an error (see PR description):
```
(node:14371) ExperimentalWarning: Type Stripping is an experimental feature and might change at any time
(Use node --trace-warnings ... to show where the warning was created)
[config] Astro found issue(s) with your configuration:

! server.allowedHosts: Did not match union.
  > Expected type { 0: string } | 
  > Received [ true ]
  ```